op-program/prestates/releases.go

```diff
@@ -0,0 +1,26 @@
+package prestates
```

Comment:
A bit of a nit: It'll be great if we can move this file to the superchain-registry somehow. There isn't any ci check or context explaining its purpose. So I can easily see this unintentionally breaking superchain-registry validation checks.

Reply:
Yea thats a good point. Although I'm not sure how to embed the json file in a go package without having a file like this with the embed directive. Do you know of a good workaround for that such that I can still read the `op-program/prestates/standards.json` content from within the superchain-registry

Reply:
I think the only option would be to have a build step that copies the file first. I'd suggest adding a simple unit test for this here and a comment that it's used in the superchain-registry.

Reply:
Added a comment. Also unit test file added here: https://github.com/ethereum-optimism/optimism/pull/12523/commits/50de20d2eb0a7d6f00c6db2f81b6e8bcd9d5861c